### PR TITLE
#248 Add multi-key capture shortcut support

### PR DESCRIPTION
**Pull request by TravonteD**
This adds the orgmode functionality documented
here: https://orgmode.org/manual/Template-elements.html

### CODE REVIEW

**Comment by TravonteD** [comment]
@kristijanhusak I've run `make format`, but the format ci still fails. Anything else that I need to do?

**Comment by TravonteD** [comment]
Got it, needed a stylua update

File: lua/orgmode/capture/init.lua [hunk 1]
```diff
@@ -18,21 +18,49 @@ function Capture:new()
...
+  end
+  return subtemplates
+end
+
+function Capture:_create_menu_items(templates)
+  local menu_items = {}
+  for key, template in pairs(templates) do
+    if string.len(key) == 1 then
+      local item = {
+        label = template.description,
```
Comment (kristijanhusak) [inline lua/orgmode/capture/init.lua:36]: label is `nil` when this is a "parent" template (string).

**Review by kristijanhusak — CHANGES_REQUESTED** [review]
Generally works ok, only that issue with the label.
I tested how Emacs does this, and basically it's same, but it has some indications that there are nested templates. Here's the gif:
nested-template
If you think those would be simple to add, that would be great. If it complicates it, I believe this implementation is also good enough.

**Comment by TravonteD** [comment]
I've added the ellipses indicator for toplevel templates as well, it was straight forward to add.

**Review by kristijanhusak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**